api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupTXNEW.java

```diff
@@ -0,0 +1,276 @@
+		SqlCohortDefinition patientWithSTARTDRUGSObs = sqlCohortQueries.getPatientWithSTARTDRUGSObs();
+		
+		// Looks for with START DATE (Concept 1190=HISTORICAL DRUG START DATE) filled in drug pickup (encounter type 18=S.TARV: FARMACIA) or follow up consultation for adults and children (encounter types 6=S.TARV: ADULTO SEGUIMENTO and 9=S.TARV: PEDIATRIA SEGUIMENTO) where START DATE is before or equal end date		
+		SqlCohortDefinition patientWithHistoricalDrugStartDateObs = sqlCohortQueries.getPatientWithHistoricalDrugStartDateObs();
+		
+		// Looks for patients enrolled on ART program (program 2=SERVICO TARV - TRATAMENTO), transferred from other health facility (program workflow state is 29=TRANSFER FROM OTHER FACILITY) between start date and end date				
+		SqlCohortDefinition transferredFromOtherHealthFacility = sqlCohortQueries.getPatientsTransferredFromOtherHealthFacility();
+		
+		CohortDefinition males = genderCohortQueries.MaleCohort();
+		
+		CohortDefinition females = genderCohortQueries.FemaleCohort();
+		
+		AgeCohortDefinition PatientBelow1Year = ageCohortQueries.patientWithAgeBelow(1);
+		AgeCohortDefinition PatientBetween1And9Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween1And9Years", 1, 9);
+		AgeCohortDefinition PatientBetween10And14Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween10And14Years", 10, 14);
+		AgeCohortDefinition PatientBetween15And19Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween15And19Years", 15, 19);
+		AgeCohortDefinition PatientBetween20And24Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween20And24Years", 20, 24);
+		AgeCohortDefinition PatientBetween25And29Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween25And29Years", 25, 29);
+		AgeCohortDefinition PatientBetween30And34Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween30And34Years", 30, 34);
+		AgeCohortDefinition PatientBetween35And39Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween35And39Years", 35, 39);
+		AgeCohortDefinition PatientBetween40And49Years = ageCohortQueries.createXtoYAgeCohort("PatientBetween40And49Years", 40, 49);
+		AgeCohortDefinition PatientBetween50YearsAndAbove = ageCohortQueries.patientWithAgeAbove(50);
+		PatientBetween50YearsAndAbove.setName("PatientBetween50YearsAndAbove");
+		
+		ArrayList<AgeCohortDefinition> agesRange = new ArrayList<AgeCohortDefinition>();
+		// agesRange.add(PatientBelow1Year);
+		// agesRange.add(PatientBetween1And9Years);
+		agesRange.add(PatientBetween10And14Years);
+		agesRange.add(PatientBetween15And19Years);
+		agesRange.add(PatientBetween20And24Years);
+		agesRange.add(PatientBetween25And29Years);
+		agesRange.add(PatientBetween30And34Years);
+		agesRange.add(PatientBetween35And39Years);
+		agesRange.add(PatientBetween40And49Years);
+		agesRange.add(PatientBetween50YearsAndAbove);
+		
+		// Male and Female <1
+		
+		CompositionCohortDefinition patientBelow1YearEnrolledInHIVStartedART = new CompositionCohortDefinition();
```

Comment:
@rubailly Can we move these composition queries into a file in the `library/cohorts` folder?

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/commit/1e2445ee70668605ab41421592b1c92b9a3b9845)